buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradleExtension.kt

```diff
@@ -115,6 +80,9 @@ open class FlankGradleExtension @Inject constructor(objects: ObjectFactory) : Fl
       name = it,
       projectId = objects.property<String>().convention(projectId),
       serviceAccountCredentials = objects.fileProperty().convention(serviceAccountCredentials),
+      debugApk = objects.property<String>().convention(debugApk),
+      instrumentationApk = objects.property<String>().convention(instrumentationApk),
+      sanityRobo = objects.property<Boolean>().convention(false),
```

Comment:
I think we shouldn't. Assuming we have `sanityRobo = true` in the base config that would propagate this to the rest, additional configs. My idea was, every config has it's own `sanityRobo` value (default false) that is independent of the rest. I am thinking about the case  when user want `runFlank` to be sanity robo test and all the rest (`runFlank[name1]`, `runFlank[name2]`) are 'normal' test runs

Of course, I can change it to behave otherwise.

Reply:
You make a good point here. I think we want consistency though. All other values are inherited. We shouldn't assume that users will not try to have multiple different `sanityRobo` configs which inherit from the base.
Let's pass the value from the base here as well.